stats/stats-server/src/main/java/ru/practicum/stats/server/model/Stats.java

```diff
@@ -0,0 +1,32 @@
+package ru.practicum.stats.server.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "stats", schema = "public")
+@Data
```

Comment:
В аннотации Data есть реализация методов equals & hashCode и это очень важные методы для сущности БД, поэтому не стоит отдавать их реализацию на откуп сторонней библиотеки ломбок, а лучше сделать их самостоятельно. В целом  достаточно вместо data ставить Getter & Setter

Подробнее здесь:
https://thorben-janssen.com/lombok-hibernate-how-to-avoid-common-pitfalls/

ключевое ниже

Your equals() method should return false if the primary key value of both entity objects is null. But Lombok’s equals() method returns true. Because of that, you can’t add two new entity objects to a Set

(для диплома требование обязательное)

Reply:
+